service/src/main/java/bio/terra/workspace/service/workspace/flight/delete/workspace/RunDeleteCloudContextFlightStep.java

```diff
@@ -98,12 +99,26 @@ public StepResult undoStep(FlightContext context) throws InterruptedException {
 
   private boolean flightComplete(FlightState flightState) {
     logger.info(
-        "Delete {} cloud context flight {} state is {}",
-        cloudPlatform,
-        flightId,
-        flightState.getFlightStatus());
+        "Delete {} cloud context state is {}", cloudPlatform, flightState.getFlightStatus());
     return (flightState.getFlightStatus() == FlightStatus.ERROR
         || flightState.getFlightStatus() == FlightStatus.FATAL
         || flightState.getFlightStatus() == FlightStatus.SUCCESS);
   }
+
+  private String getFlightId(FlightContext context, CloudPlatform cloudPlatform) {
+    // Retrieve our flight id
+    Map<CloudPlatform, String> flightIds =
+        context.getWorkingMap().get(WorkspaceFlightMapKeys.FLIGHT_IDS, new TypeReference<>() {});
```

Comment:
getrequred

Reply:
oh sorry didn't know that there isn't one.

Reply:
NP 
It is trivial.